### DISCUSSION TITLE
schedule not respected

### DISCUSSION THREAD

**Discussion by oprogramador** [renovatebot/renovate#22838] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

In https://github.com/oprogramador/array-starts-with/blob/master/renovate.json, I have `"schedule": ["after 11pm on sunday"],`. However, a few minutes ago (Friday in UTC) I just got a new commit force-pushed in this PR: https://github.com/oprogramador/array-starts-with/pull/35 .

Does `schedule` work only for new PRs? My goal is to get notifications from Renovate as rare as possible. Is it possible to configure Renovate to open/update PRs on monthly/quarterly or yearly basis?

I don't have any minimum reproduction repo as I don't have enough time, and I'm not sure what's the expected behavior.

**Comment by rarkins** [comment, marked as answer]
Expected behavior. See https://docs.renovatebot.com/presets-default/#updatenotscheduled

**Comment by oprogramador** [comment]
Thanks for your reply.

In https://docs.renovatebot.com/presets-schedule/ , I can see e.g. `every 3 months on the first day of the month`. Would it work to have `every 3 months on the first Saturday of the month` (or another day of week)?

And where can I see the nearest expected scheduled time to predict when it'll run (open a PR or rebase or merge)?

Thanks in advance.

**Reply by rarkins** [reply]
I recommend you use cron syntax for scheduling for anything complex

**Reply by oprogramador** [reply]
Is there any documentation for the cron syntax?

And you haven't replied regarding checking predicted next run time.

**Reply by rarkins** [reply]
We don't provide predictions

**Reply by rarkins** [reply]
https://crontab.guru/crontab.5.html
